static/script/devices/mediaplayer/mediaplayer.js

```diff
@@ -312,6 +326,11 @@ require.def(
                 } else {
                     return this.CLAMP_OFFSET_FROM_END_OF_RANGE;
                 }
+            },
+
+            _isLiveMedia: function () {
+                return (this._type === MediaPlayer.TYPE.LIVE_VIDEO)
+                    || (this._type === MediaPlayer.TYPE.LIVE_AUDIO);
```

Comment:
JsHint will moan about bad line breaking here - avoid by putting the || on the previous line (or by putting it all on the same line).